pkg/db/src/implementations/version-1.2/customer-orders.ts

```diff
@@ -0,0 +1,168 @@
+import { BehaviorSubject, firstValueFrom, Observable, ReplaySubject, share, Subject } from "rxjs";
+
+import { VolumeStock } from "@librocco/shared";
+
+import { DocType } from "@/enums";
+
+import { VersionedString, OrdersDatabaseInterface, CustomerOrderInterface, CustomerOrderData } from "@/types";
+
+import { versionId } from "./utils";
+import { isEmpty, isVersioned, runAfterCondition, uniqueTimestamp } from "@/utils/misc";
+import { newDocumentStream } from "@/utils/pouchdb";
+
+
+class CustomerOrder implements CustomerOrderInterface {
+
+	// {
+	// 	"id": string, // use 'timestampedId' (or 'timestampId', not sure, it's in the 'db' package, under utils I think
+	// 	"draft": boolean,
+	// 	"displayName": string, // "name" -> "displayName" to keep consistent with the rest of the app
+	// 	"email": string,
+	// 	"books": [ 
+	// 	  { isbn: string }, // No status for now we can add that later when dealing with supplier orders
+	// 	],
+	// 	"deposit"?: number // I think the deposit should be calculated on display without persisting and then added when the order is placed
+	// 	}
```

Comment:
Let's remove all excess comments. I understand you've used them as helper (which makes total sense), but I think the code that ends up in `main` should be clean.